src/main/java/fr/ospiea/oscra/user/controller/UserController.java

```diff
@@ -1,47 +0,0 @@
-package fr.ospiea.oscra.user.controller;
-
-import fr.ospiea.oscra.user.object.User;
-import fr.ospiea.oscra.user.service.UserService;
-import io.swagger.annotations.*;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.*;
-
-import java.util.List;
-
-/**
- * Created by fqin on 18/09/2016.
- */
-@RestController
-@RequestMapping("/users")
```

Comment:
I agree with the logic. But Spring security has already a system of authentication and authorization, in addition, the class User has already been implemented in Spring Security core. So in order to avoid conflict with the pre-defined class, I had changed the class name. Maybe Employee is not a suitable name here neither, anyway in my point of view, we should not use class name "user", maybe "OspieaUser", "account" or any thing else.